pynest/nest/lib/hl_api_simulation.py

```diff
@@ -39,6 +39,49 @@ def Simulate(t):
     sps(float(t))
     sr('ms Simulate')
 
+@check_stack
+def Run(t):
+    """Simulate the network for t milliseconds.
+
+    Parameters
+    ----------
+    t : float
+        Time to simulate in ms
+    """
+
+    sps(float(t))
+    sr('ms Run')
+
+@check_stack
+def Prepare():
+    """Simulate the network for t milliseconds.
+
+    Parameters
+    ----------
+    t : float
+        Time to simulate in ms
+    """
+
+    sr('Prepare')
+
+
+@check_stack
+def Cleanup():
+    """Simulate the network for t milliseconds.
+
+    Parameters
+    ----------
+    t : float
+        Time to simulate in ms
+    """
+
+    sr('Cleanup')
+
+@contextmanager
+def IterateRuns():
+    Prepare()
+    try: yield
+    finally: Cleanup()
```

Comment:
The name is wrong -- it's not iterate runs, it's manage runs. It should also be commented as to use:
```
with nest.IterateRuns():
  for i in range(0, 10):
    nest.run(5)
```

It's the messed up fix for a lack of proper lambda's in python. See `contextlib.contextmanager`.